sdk/maps/maps-search/review/maps-search.api.md

```diff
@@ -0,0 +1,756 @@
+export type ReverseSearchAddressBatchOptions = OperationOptions & BatchPollerOptions;
+
+// @public
+export interface ReverseSearchAddressOptions extends ReverseSearchBaseOptions {
+    allowFreeformNewline?: boolean;
+    entityType?: GeographicEntityType;
+    includeMatchType?: boolean;
+    includeRoadUse?: boolean;
+    includeSpeedLimit?: boolean;
+    roadUse?: RoadUseType[];
+    streetNumber?: string;
+}
+
+// @public
+export interface ReverseSearchAddressRequest {
+    // (undocumented)
+    coordinates: LatLon;
+    // (undocumented)
+    options?: ReverseSearchAddressOptions;
+}
+
+// @public (undocumented)
+export interface ReverseSearchAddressResult {
+    readonly numResults: number;
+    readonly queryTime: number;
+    readonly results: ReverseSearchAddressResultItem[];
+}
+
+// @public (undocumented)
+export interface ReverseSearchAddressResultItem {
+    readonly address: Address;
+    readonly matchType?: MatchType;
+    readonly position: LatLon;
+    readonly roadUse?: RoadUseType[];
+}
+
+// @public
+export interface ReverseSearchBaseOptions {
+    heading?: number;
+    language?: string;
+    localizedMapView?: LocalizedMapView;
+    radiusInMeters?: number;
+}
+
+// @public
+export interface ReverseSearchCrossStreetAddressOptions extends ReverseSearchBaseOptions, OperationOptions {
+    top?: number;
+}
+
+// @public (undocumented)
+export interface ReverseSearchCrossStreetAddressResult {
+    readonly numResults: number;
+    readonly queryTime: number;
+    readonly results: ReverseSearchCrossStreetAddressResultItem[];
+}
+
+// @public (undocumented)
+export interface ReverseSearchCrossStreetAddressResultItem {
+    readonly address?: Address;
+    readonly position?: LatLon;
+}
+
+// @public
+export type RoadUseType = string;
+
+// @public
+export interface SearchAddressBaseOptions extends SearchBaseOptions {
+    boundingBox?: BoundingBox;
+    isTypeAhead?: boolean;
+    radiusInMeters?: number;
+}
+
+// @public
+export type SearchAddressBatchOptions = OperationOptions & BatchPollerOptions;
+
+// @public
+export interface SearchAddressOptions extends SearchAddressBaseOptions {
+    coordinates?: LatLon;
+    countryCodeFilter?: string[];
+    entityType?: GeographicEntityType;
+}
+
+// @public
+export interface SearchAddressRequest {
+    // (undocumented)
+    options?: SearchAddressOptions;
+    // (undocumented)
+    query: string;
+}
+
+// @public
+export interface SearchAddressResult {
+    readonly fuzzyLevel?: number;
+    readonly geoBias?: LatLon;
+    readonly numResults: number;
+    readonly query?: string;
+    readonly queryTime: number;
+    readonly queryType?: QueryType;
+    readonly results: SearchAddressResultItem[];
+    readonly skip?: number;
+    readonly totalResults?: number;
+}
+
+// @public
+export interface SearchAddressResultItem {
+    readonly address: Address;
+    readonly addressRanges?: AddressRanges;
+    readonly dataSources?: DataSource;
+    readonly detourTime?: number;
```

Comment:
What's the unit of `detourTime`? Would be better to include the unit in the property name

Reply:
b5390e30a813e59d894775903b3c7e1dc6c447da